django/core/management/commands/loaddata.py

```diff
@@ -163,7 +168,7 @@ def load_label(self, fixture_label):
 
                 objects = serializers.deserialize(
                     ser_fmt, fixture, using=self.using, ignorenonexistent=self.ignore,
-                )
+                    handle_forward_references=True)
```

Comment:
There  should be a trailing comma and the parenthesis goes on the next line as it was before.